yarn.lock

```diff
@@ -8475,6 +8520,61 @@ elliptic@^6.0.0, elliptic@^6.4.0, elliptic@^6.4.1:
     minimalistic-assert "^1.0.0"
     minimalistic-crypto-utils "^1.0.0"
 
+embark-core@^5.1.1-nightly.2:
```

Comment:
Whenever `embark[js][-*]` shows up in the `yarn.lock` it means some dev/Deps range/s need to be adjusted. See: https://github.com/embarklabs/embark/blob/master/scripts/check-yarn-lock.js.